llmware/library.py

```diff
@@ -583,7 +857,48 @@ def install_new_embedding (self, embedding_model_name=None, vector_db=None,
                                from_hf= False, from_sentence_transformer=False, model=None, tokenizer=None, model_api_key=None,
                                vector_db_api_key=None, batch_size=500, max_len=None, use_gpu=True):
 
-        """ Main method for installing a new embedding on a library """
+        """ Main method for installing a new embedding on a library 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Main method for installing a new embedding on a library.
```